docs/app/pages/components/sections/panels/item-display-panel/item-display-panel.component.ts

```diff
@@ -15,10 +15,7 @@ import { ItemDisplayPanelComponent } from '../../../../../../../src/index';
 export class ComponentsItemDisplayPanelComponent implements IPlunkProvider {
```

Comment:
All looks good the only thing I did notice was the content of the display panel in plunker seems a bit off:

![image](https://cloud.githubusercontent.com/assets/20795331/26624636/61dea5fc-45e9-11e7-8e49-9133a1bf7c3c.png)